tpgtools/README.md

```diff
@@ -176,3 +176,33 @@ doc_hide:
 test_hide:
   - basic_workload.yaml
 ```
+
+## New Resource Guide (tpgtools only)
+
+This guide is written to document the process for adding a resource to the Google Terraform Provider (TPG) after it has been added to the [DCL](https://github.com/GoogleCloudPlatform/declarative-resource-client-library).
+
+### Adding Resource Overrides
```

Comment:
For this + the header below: I'd recommend assuming that the override file will _always_ be blank, explain briefly what an override is, and recommend that folks reach out to an experienced team member if they believe an override will be needed for their resource(s). To cover brownfield resources, I'd leave them out of this flow entirely and add them as an appendix afterwards- "those were the guidelines for new resources, here's how a brownfield resource is different"

Motivation(s):
* This keeps things simple for folks that are making their first contribution. It isn't true that you never need overrides yet, but we need to make it true.
* Deferring to an experienced engineer will mean they can help solve the root cause that needed an override in the first place.
* This is in contrast to the MMv1 field guide, where I explain some MMv1 overrides- using them is pretty normal, even for simple contributions.
* We expect most brownfield conversions to be done for our team (or another experienced member of a sister team) so having the information out of a new dev's critical path means there's less information overload